config/config.go

```diff
@@ -73,20 +74,22 @@ type (
 		}
 
 		DigitalOcean struct {
-			Token    string
-			Image    string
-			UserData string `split_words:"true"`
-			Region   string
-			SSHKey   string
-			Size     string
-			Tags     []string
+			Token        string
+			Image        string
+			UserData     string `split_words:"true"`
+			UserDataPath string `env:"DRONE_DIGITALOCEAN_USERDATA_PATH"`
```

Comment:
I think we might need to remove `split_works` and specify the full environment variable name.

Reply:
I have not defined split words for this variable :(